ui/admin/tests/helpers/index.js

```diff
@@ -18,19 +19,19 @@ function setupApplicationTest(hooks, options) {
   upstreamSetupApplicationTest(hooks, options);
 
   // Additional setup for application tests can be done here.
-  //
-  // For example, if you need an authenticated session for each
-  // application test, you could do:
-  //
+
+  setupMirage(hooks);
+
   hooks.beforeEach(async function () {
-    await authenticateSession();
+    this.server.create('scope', { id: 'global' }, 'withGlobalAuth');
+    const account_id = this.server.schema.accounts.first().id;
+    await authenticateSession({ account_id });
```

Comment:
Thoughts on refactoring our test cases to use this helper in order to provide an account_id  to the `authenticateSession`? I only pushed up changes for `tests/acceptance/accounts` as a POC.